includes/wikia/models/DesignSystem/WdsExternalImage.class.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class WdsExternalImage {
```

Comment:
how about having all data that is to be serialized in public vars?
```
class MyClass {
    public $url = 'my-url';
    public $type = 'image-external';
}

$m = new MyClass();

echo json_encode($m);
```

These classes should be much simplier then and we would not have to have get() - method, which I find ugly. as you have a class to represent some data but then inside that class you have to do it once again in get method.

What do you think?

@kvas-damian you are PHP master here - does that make sense?